browser/net/brave_proxying_web_socket.cc

```diff
@@ -151,35 +151,6 @@ void BraveProxyingWebSocket::OnOpeningHandshakeStarted(
   forwarding_handshake_client_->OnOpeningHandshakeStarted(std::move(request));
 }
 
-void BraveProxyingWebSocket::OnResponseReceived(
```

Comment:
cc @iefremov

Reply:
OnConnectionEstablished is called instead; so this should be good to go 👍